messages/control/command.json

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Command",
+    "description": "Command to an actuator.",
+    "default": {},
+    "required": [
+        "target"
+    ],
+    "properties": {
+        "target": {
+            "type": "object",
+            "title": "Target",
+            "description": "Target value or values for the actuator.",
+            "minProperties": 1
+        },
+        "additionalProperties": true
+    }
+}
```

Comment:
This is very generic and we dont have a clear usecase for it yet. I suggest we postpone this for later.